.gitignore

```diff
@@ -125,7 +125,7 @@ report.asciidoc
 # Yarn local mirror content
 .yarn-local-mirror
 
-# Bazel
+# Bazel | TODO: Remove later
```

Comment:
let's only remove these a bit later - if might cause a lot of inconvenience if there are folders in and out of git ignore when devs change branches around.

Reply:
👍